promotion/datastore.go

```diff
@@ -47,6 +47,7 @@ type Datastore interface {
 // Postgres is a Datastore wrapper around a postgres database
 type Postgres struct {
 	*sqlx.DB
+	location string
```

Comment:
location is not descriptive, also it doesn't seem very useful to have it as part of this struct.

how about just passing it as an argument to `Migrate` / `NewMigrate`? perhaps we can make it optional and lookup from an env var there (in the pattern of what we're doing for `databaseURL` in `NewPostgres`?